_fastkit.code-workspace

```diff
@@ -282,6 +282,9 @@
     }
   ],
 	"settings": {
+    "search.followSymlinks": false,
+    "search.maxResults": 5000,
+    "search.searchOnType": false,
```

Comment:
Certainly there are no use cases where you want to get more than 5,000 hits, and the development experience is more valuable than the accuracy of 5,000+ hits.